drivers/hmis/app/models/hmis/hud/household.rb

```diff
@@ -102,6 +114,18 @@ def self.sort_by_option(option)
         hh_t[:latest_exit].eq(nil).desc,
         earliest_entry: :desc,
       )
+    when :hoh_last_name_a_to_z
+      joins(:enrollments).where(e_t[:RelationshipToHoH].eq(1)).joins(:clients).order(c_t[:LastName].asc.nulls_last)
```

Comment:
Does this join look correct? Simply `joins(:clients)` also seems to work fine, just by experimentation, but thought maybe that was relying on some non-guaranteed sort order of the clients.

Reply:
That's probably worth enforcing the sort order as you did, but I'd use a merge instead of arel for the HoH clause.
```suggestion
      joins(:enrollments, :client).
      merge(Hmis::Hud::Enrollment.heads_of_households).
      order(c_t[:LastName].asc.nulls_last)
```

Reply:
This is nice! Thank you